CONTRIBUTING.rst

```diff
@@ -1,177 +1,228 @@
+    - Template: `Concept article template`_
 
+* **Reference** starts with an overview of what can be found in this section, and then usually a list of some kind. We use this type of article for lists of extensions, configuration parameters, and that sort of bookmark-it-for-reference type of content.
+    - Example: https://developer.aiven.io/docs/products/kafka/reference/advanced-params.html
 
-**What is the content model?**
+Pull request process
+--------------------
 
-DevPortal contains the following content types:
-- *Concept* - Conceptual content helps people understand a feature or topic by providing a clear, high-level overview, explanation of how the feature or topic can help them on their journey, and context like use cases or examples. 
-- *Task* - Task content helps people complete a task from start to finish while they're using Aiven’s products.
-- *Reference* - Referential content provides detailed information that people need while they're actively using Aiven's products.
-- *Tutorial* - Tutorials help people learn about products, and solve real world problems by guiding them through the entire workflow to complete a task.
+The instructions for setting up local development are in the ``README``.
 
-.. tip::
-    Use the templates included below for further guidance. 
+0. We heartily recommend you open an issue so that we can either help with outlines, or know that three people are working on the same thing.
 
-To learn more about documentation content types, see the `Diátaxis Framework <https://diataxis.fr/>`_.
+1. Make your changes on a branch. We recommend a maximum of three articles, and don't review large changesets without prior discussion. File names are ``kebab-case``, with hyphens, such as ``connect-with-pgadmin.rst``. Files should also be added to the table of contents file ``_toc.yml`` so they appear in the right location in the navigation.
 
+2. Open a pull request, use the template so we know what it is and why it's a great addition. If you need to publish this quickly for a reason, mention that as well.
 
-**What is the structure of DevPortal?**
+3. We have an automated build process that checks your work against our in-house styles, gets product names with the right capitalization, and checks links. Please don't be alarmed if your build fails! View the output of the failed jobs to find out what happened, or we can advise when we review. You can run these locally if you like: ``make spell`` and ``make linkcheck``.
 
-You can see the structure in the left-hand navigation menu. 
-It is driven by a plugin called `Sphinx external TOC <https://sphinx-external-toc.readthedocs.io/en/latest/intro.html>`_. 
+.. tip::
 
-You can find the current structure in ``_toc.yml``.
+    Don't add commands to the accepted words list, these should always be in ``literal`` markup and ignored by the spell checker. Literal markup can't be used inside link text, so reword your link and sentence to avoid this.
```

Comment:
Sometimes they have to add the new product names, right? I saw in a commit just now **Slack**, it doesn't fit ``literal`` markup, so such terms need to be added.